providers/celery/tests/unit/celery/executors/test_celery_executor.py

```diff
@@ -126,6 +126,17 @@ def test_supports_sentry(self):
     def test_cli_commands_vended(self):
         assert CeleryExecutor.get_cli_commands()
 
+    def test_celery_executor_init_with_args_kwargs(self):
```

Comment:
I guess we should add something like this here so this test doesn't run on older versions:
```suggestion
    @pytest.mark.skipif(not AIRFLOW_V_3_1_PLUS, reason="Multi-team support was added to Celery constructor in 3.1.0")
    def test_celery_executor_init_with_args_kwargs(self):
```

Reply:
Yupp! I ran the tests locally, but those don't run the full release history tests of course. Will fix this up!